lib/ExpensiMark.js

```diff
@@ -250,4 +267,23 @@ export default class ExpensiMark {
 
         return replacedText;
     }
+
+    /**
+     * Replaces html with elements
+     *
+     * @param {String} htmlString
+     *
+     * @returns {String}
+     */
+    htmlToMarkdown(htmlString) {
+        let generatedMarkdown = htmlString;
+        this.htmlToMarkdownRules.forEach((rule) => {
+            // Pre-processes input HTML before applying regex
```

Comment:
Please add an empty line before this comment (like the one above).

Reply:
![Screenshot from 2021-06-05 20-53-10](https://user-images.githubusercontent.com/32242596/120896630-10faa380-c640-11eb-89a6-9e4e2a8cbe4c.png)

Reply:
Ah, yea, that's due to the whitespace on that line. If you backspace a bunch of times it'll get rid of that linting error.

Reply:
Did this get resolved?

Reply:
I tried but it's not working 

![Screenshot from 2021-06-15 22-53-26](https://user-images.githubusercontent.com/32242596/122096769-89294c00-ce2c-11eb-82e5-73c833fd1c85.png)

Reply:
Hmm, OK. Not a big deal I guess. Thanks for trying